### DISCUSSION TITLE
no one is publishing on "mystream"

### DISCUSSION THREAD

**Discussion by ilpiruz** [bluenviron/mediamtx#1492] [General]
i've put http://loaclhost:8888/mystream but everytime i open via vlc, the rtsp server logs "destroyed (no one is publishing on mystream)

**Comment by kdill00** [comment]
post the paths section of your config. Also you have localhost misspelled in the comment if you copied and pasted.

**Reply by ilpiruz** [reply]
> post the paths section of your config. Also you have localhost misspelled in the comment if you copied and pasted.

i already fixed. somehow it worked after 5 minutes
thanks anyway

[a comment by an automated account is not shown]
